oresat_c3/services/edl.py

```diff
@@ -85,84 +89,99 @@ def __init__(
         self._edl_rejected_count_obj = edl_rec["rejected_count"]
         self._last_edl_obj = edl_rec["last_timestamp"]
 
-    def _get_hmac_key(self) -> bytes:
-        """GEt the active HMAC key."""
-
+    @property
+    def _hmac_key(self) -> bytes:
         edl_rec = self.node.od["edl"]
         active_key = edl_rec["active_crypto_key"].value
         return edl_rec[f"crypto_key_{active_key}"].value
 
-    def _upack_last_recv(self) -> Union[EdlPacket, None]:
-        req_packet = None
+    @property
+    def _flight_mode(self) -> bool:
+        return bool(self._flight_mode_obj.value)
 
-        if len(self._radios_service.recv_queue) == 0:
-            return req_packet
+    @property
+    def _sequence_count(self) -> int:
+        return self._edl_sequence_count_obj.value
 
-        req_message = self._radios_service.recv_queue.pop()
+    @_sequence_count.setter
+    def _sequence_count(self, value):
+        self._edl_sequence_count_obj.value = value
 
+    @property
+    def _rejected_count(self) -> int:
+        return self._edl_rejected_count_obj.value
+
+    @_rejected_count.setter
+    def _rejected_count(self, value):
+        self._edl_rejected_count_obj.value = value
+
+    def _upack_last_recv(self) -> Optional[EdlPacket]:
```

Comment:
Some amount of terseness does help with readability and it did shorten a couple lines enough to save them from being mangled by `black`. More importantly though there is exactly one correct way for this class to use the `ODVariables` (through the `value` field) and a whole bunch of ways for this class to incorrectly use them (touching literally anything else). Instead of leaving open the opportunity to misuse them, the properties ensure they're used correctly, every time.